src/class-wpml-gutenberg-integration-factory.php

```diff
@@ -31,27 +31,20 @@ public function create() {
 			)
 		);
 
-		$reusable_blocks             = new WPML\PB\Gutenberg\ReusableBlocks\Blocks();
-		$reusable_blocks_translation = new WPML\PB\Gutenberg\ReusableBlocks\Translation( $sitepress );
-
-		$integrations->add(
-			new WPML\PB\Gutenberg\ReusableBlocks\Integration( $reusable_blocks_translation )
+		$is_reusable_block_translatable = $sitepress->is_translated_post_type(
+			WPML\PB\Gutenberg\ReusableBlocks\Translation::POST_TYPE
 		);
 
-		if ( is_admin() ) {
+		if ( $is_reusable_block_translatable ) {
```

Comment:
@strategio I think it would be better:

`if( $this->should_translate_reusable_blocks() ) {`

Reply:
Good idea @brucepearson, done!